src/Options/default_options.jl

```diff
@@ -126,6 +128,9 @@ function validate_solver_options_and_add_default_values!(options::Options)::Opti
             domain_constraints = (v::Int  ->  v > 0)
         elseif key == :transformation_matrix
             expected_type = Any
+        elseif key == :global_vars
+            expected_type = Vector{Int}
+            domain_constraints = (v::Vector{Int}  ->  length(v) >= 0)
```

Comment:
I guess initially it was `>` and then I changed it to `>=` which makes this code snippet useless. Actually your check makes more sense, because we can just ignore adding `global_vars` and then we should get results only with constrained variables.